cmd/admin-server-info.go

```diff
@@ -111,3 +113,38 @@ func getLocalDrivesPerf(endpoints EndpointList, size int64, r *http.Request) mad
 		Perf: dps,
 	}
 }
+
+// getLocalCPUHardware - returns ServerCPUHardwareInfo only for the
+// local endpoints from given list of endpoints
+func getLocalCPUHardware(endpoints EndpointList, r *http.Request) ServerCPUHardwareInfo {
+	var cpuHardwares []cpuinfo.InfoStat
+	scratchSpace := map[string]bool{}
+	for _, endpoint := range endpoints {
+		if !endpoint.IsLocal {
```

Comment:
This change should look at the changes in other getLocal* implementations endpoint.Host should be skipped first.

Reply:
done.